assembl/views/api/synthesis.py

```diff
@@ -27,7 +27,7 @@ def get_syntheses(request):
         raise HTTPNotFound("Discussion with id '%s' not found." % discussion_id)
     user_id = authenticated_userid(request) or Everyone
     permissions = get_permissions(user_id, discussion_id)
-    syntheses = discussion.get_all_syntheses()
+    syntheses = discussion.get_all_syntheses_query()
```

Comment:
Right, remind me again, is the sqla `query` object an iterator that you can do comprehensions on, which would then do a `query.all()` behind the scenes?